### ISSUE TITLE
G6 - Low contrast text if background image fails to load

### ISSUE THREAD

**Issue by mmatthews** [ccswbs/hjckrrh#286]
If the background image on `#ug-header` fails to load, the header's white text will not have sufficient contrast against the element's white background.
This could be fixed by giving `#ug-header` a background-color as a fallback. e.g.

    background-color: #111;

**Comment by mancusop** [comment]
@mmafe 
request to merge this fix in next sprint?

**Comment by mancusop** [comment]
As per @mmafe request promoting this to high priority. Updates on the issue -

- If Images are turned off G6 becomes visually un-navigable.
- This is considered a faliure by WCAG 2.0 AA standards i.e. - F24: Failure of Success Criterion 1.4.3, 1.4.6 and 1.4.8 due to specifying foreground colors without specifying background colors or vice versa.
- UoG Main Page has corrected and fixed this issue but AODA code has not.

AODA:
![no_images_aoda](https://cloud.githubusercontent.com/assets/15733616/13475539/8d1ad61e-e091-11e5-882b-030eecbf7c19.png)

UoG Home:
![no_images_uog](https://cloud.githubusercontent.com/assets/15733616/13475546/9e820dc8-e091-11e5-9582-a1ccf2aabeaf.png)


More info on F24 - Faliure of SC: https://www.w3.org/TR/WCAG20-TECHS/F24.html

**Comment by mancusop** [comment]
Enhancement has been implemented https://github.com/ccswbs/hjckrrh/pull/361
